### PR TITLE
Update rules_cc

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

**Comment by keith** [comment]
@oquenchil can you review 🙏🏻

**Comment by ted-xie** [comment]
Looks like src/test/shell/bazel/android/android_integration_test.sh was the only Android-related file changed, and the change looks good to me.

**Review by ted-xie — APPROVED** [review]
LGTM, though let's wait for @oquenchil to also review.

File: MODULE.bazel [hunk 1]
```diff
@@ -10,12 +10,12 @@ bazel_dep(name = "rules_license", version = "0.0.3")
...
 bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
 bazel_dep(name = "grpc", version = "1.47.0", repo_name = "com_github_grpc_grpc")
-bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "platforms", version = "0.0.6")
 bazel_dep(name = "rules_pkg", version = "0.7.0")
 bazel_dep(name = "stardoc", version = "0.5.3", repo_name = "io_bazel_skydoc")
 bazel_dep(name = "zstd-jni", version = "1.5.2-3")
 bazel_dep(name = "zlib", version = "1.2.13")
-bazel_dep(name = "rules_cc", version = "0.0.2")
+bazel_dep(name = "rules_cc", version = "0.0.5")
```
Comment (brentleyjones) [inline MODULE.bazel:18]: 0.0.6 now?

Comment (keith) [reply MODULE.bazel:18]: oops thanks

**Review by oquenchil — APPROVED** [review]

**Pull request closed without merging**